lib/puppet/provider/java_ks/keytool.rb

```diff
@@ -169,7 +169,11 @@ def latest
         '-storetype', 'PKCS12', '-storepass', sourcepassword
       ]
       output = run_command(cmd)
-      latest = output.scan(%r{\(SHA1\):\s+(.*)|\(MD5\):\s+(.*)})[0][0]
+      latest = if output.include? 'MD5:'
```

Comment:
Maybe worth refactoring this to be a utility method like 'extract_fingerprint', as the same logic is used by both `current` and `latest`.